src/main/java/com/zeepy/server/review/service/ReviewService.java

```diff
@@ -0,0 +1,39 @@
+package com.zeepy.server.review.service;
+
+import com.zeepy.server.common.CustomExceptionHandler.CustomException.NoContentException;
+import com.zeepy.server.review.domain.Review;
+import com.zeepy.server.review.dto.ReviewResponseDto;
+import com.zeepy.server.review.dto.ReviewResponseDtos;
+import com.zeepy.server.review.dto.ReviewDto;
+import com.zeepy.server.review.repository.ReviewInterface;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class ReviewService {
+    private final ReviewInterface reviewInterface;
+
+    @Transactional(readOnly = true)
+    public ReviewResponseDtos getReviewList(String address) {
+        List<Review> reviewList = reviewInterface.findAllByAddress(address);
+        if(reviewList.isEmpty()) ThrowNoContentException();
+        return new ReviewResponseDtos(reviewList.stream()
+                .map(ReviewResponseDto::new)
+                .collect(Collectors.toList()));
```

Comment:
```suggestion
        List<Review> reviewList = reviewInterface.findAllByAddress(address);
        if(reviewList.isEmpty()) {
            throw new NoContentException();
        }
        return new ReviewResponseDtos(reviewList);
```
불필요하게 메소드를 만들 필요는 없고, 생성자를 다양하게 만들어주면 좋습니다.